digdag-standards/src/main/java/io/digdag/standards/operator/td/TdLoadOperatorFactory.java

```diff
@@ -104,7 +108,7 @@ else if (config.isPresent()) {
                 this.sessionName = Optional.absent();
             }
             else {
-                throw new AssertionError();
+                throw new ConfigException("Unexpected config error");
```

Comment:
We already have `numExistParams <= 0` that you added in this PR so, I think that we don't need to the `else` statement any more. or, 2nd `else if` can be changed with `else`? What do you think?

```
else { // config.isPresent()
    this.embulkConfig = Optional.of(config.get().getInternalObjectNode());
    this.sessionName = Optional.absent();
}
```

Reply:
Yes, I also think this code may never reached. I will fix it with your proposal. Thanks!